packages/syft/src/syft/core/tensor/autodp/phi_tensor.py

```diff
@@ -1398,6 +1398,28 @@ def transpose(self, *args: Any, **kwargs: Any) -> PhiTensor:
             max_vals=max_vals,
         )
 
+    def flatten(self, order: Optional[str] = "C") -> PhiTensor:
+        product = 1
+        for i in self.min_vals.shape:
+            product *= i
+        output_shape = (product,)
+        print(self.min_vals.data, self.min_vals.data.shape)
```

Comment:
@IshanMi It looks like we left a `print` statement in here 😬

Reply:
Great catch! Removed :+1: